sdk/messaging/azeventhubs/consumer_client.go

```diff
@@ -39,31 +39,40 @@ const (
 
 // ConsumerClientOptions configures optional parameters for a ConsumerClient.
 type ConsumerClientOptions struct {
-	// TLSConfig configures a client with a custom *tls.Config.
-	TLSConfig *tls.Config
-
-	// Application ID that will be passed to the namespace.
+	// ApplicationID is used as the identifier when setting the User-Agent property.
 	ApplicationID string
 
+	// Identifier is a unique name used to identify the consumer. This can help with
+	// diagnostics as this name will be returned in error messages. By default,
+	// an identifier will be automatically generated.
+	Identifier string
```

Comment:
What if we called this `InstanceID` instead (I made the same note on the APIView).  I think this helps separate it from `ApplicationID` and clarifies the usage.

Reply:
I like that. 

There's a corresponding `ID()` function on ConsumerClient that we should change to match.

Reply:
https://apiview.dev/Assemblies/Review/8245669789dd4453bea087a9afaf2705?diffRevisionId=6ec7cf3099724b8bad89e1ce538b32d9&diffOnly=False&revisionId=c445bbc6810a4e2e95565b97f7e72434&doc=False